model/Hierarchy.php

```diff
@@ -218,10 +218,12 @@ public function getChildrenAsUL($attributes = "", $titleEval = '"<li>" . $child-
 	 * @param int $nodeCountThreshold See {@link getChildrenAsUL()}
 	 * @return int The actual number of nodes marked.
 	 */
-	public function markPartialTree($nodeCountThreshold = 30, $context = null,
+	public function markPartialTree($nodeCountThreshold = null, $context = null,
 			$childrenMethod = "AllChildrenIncludingDeleted", $numChildrenMethod = "numChildren") {
 
-		if(!is_numeric($nodeCountThreshold)) $nodeCountThreshold = 30;
+		if(!is_numeric($nodeCountThreshold)) {
+			$nodeCountThreshold = Config::inst()->get('Hierarchy', 'node_threshold_total');
```

Comment:
Can we use `$this->owner->config()->get('node_threshold_total')` so that we can configure these on a per-object basis?